client/tailwind.config.js

```diff
@@ -1,12 +1,8 @@
-// @type {import('tailwindcss').Config} 
+/** @type {import('tailwindcss').Config} */
```

Comment:
If it's not needed, you should delete it

Reply:
ok